### DISCUSSION TITLE
How to detect the tabular structure in a image

### DISCUSSION THREAD

**Discussion by Shine-5705** [PaddlePaddle/PaddleOCR#13836] [Q&A]
I encountered an issue while using PaddleOCR for table structure recognition on the attached image: [image00003]![image00003](https://github.com/user-attachments/assets/5537ac47-1e1e-4530-b6cc-c33704a92030)

When I tried to extract the table data, the output was not as expected. Here are the key issues I observed:

**Overlapping Rows and Columns**: The extracted CSV file ([improved_extracted3_table.csv](https://github.com/user-attachments/files/16907656/improved_extracted3_table.csv)) shows overlapping rows and columns, making the data difficult to interpret.

**Incomplete Text Detection**: Not all the text within the table cells was detected. Some data is missing, which affects the accuracy of the extracted information.

Given these issues, could anyone please suggest possible solutions or improvements to enhance the table structure recognition for this alike image? Any advice on how to better handle complex table structures with overlapping elements would be greatly appreciated.

Thank you for your support!

**Comment by Bishow-99** [comment]
@Shine-5705 you need to finetune the model on you custom dataset because PaddleOCR can't perfectly work on complex table that are borderless as well as contain the multiple lines text of the same celll

**Comment by CapturefastCEO** [comment]
Can you try the image by using the Live Demo link and Table Detection function on https:\\BeyondOCR.com
